parm/ocnicepost/ice_mx025_to_0p25.nml

```diff
@@ -0,0 +1,11 @@
+& ocnicepost_nml
+ftype='ice'
+wgtsdir='/scratch1/NCEPDEV/stmp4/Denise.Worthen/CPLD_GRIDGEN/'
```

Comment:
This should not be hardcoded. A template (jinja) should be used as part of the GW to create this file.

Reply:
This is just a sample of what @DeniseWorthen provided on Hera.  
The workflow will use the jinja template to fill it properly.  
Once we write a test for this program, these sample files will be removed.

Reply:
See `ocnicepost.nml.jinja2` for the template.